src/lib/api/group/client/groupApiClient.client.test.ts

```diff
@@ -32,27 +38,66 @@ describe('API Function Tests', () => {
     jest.clearAllMocks();
   });
 
-  // createGroup method tests
-  describe('createGroup', () => {
-    // mock params and response
-    const mockCreateGroupParams: ICreateGroupParams = { groupName: 'Shared-house' };
-    const mockCreateGroupResponse: ICreateGroupApiResponse = {
-      groupId: 'a3kdifut-a520-c2cb-1be7-d90710691861',
-    };
+  describe('postCreateGroup', () => {
+    // Arrange mock data
+    const mockPayload: IPostCreateGroupPayload = { groupName: 'Shared-house' };
     it('successfully creates a group', async () => {
```

Comment:
Could you add a line break before each `it()` and `describe` to improve readability, please?

Reply:
Thank you for fixing it, but I found one in the `src/features/groups/lib/actions.test.ts`, too.